### PR TITLE
Range sliders for degree objectives 

### PR DESCRIPTION
**Pull request by juliachaloupka**
This PR implements the functionality to add range sliders for degree objective instead of select boxes on edit profile view.

## Issue Related
#200 

## Type of change
Changed select boxes to range sliders for degree objective on edit profile view 

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Front end change

## Code coverage
 93.22%

## Screenshot (if any change in front end)
Screenshot 2019-03-28 01 09 33

### CODE REVIEW

File: app/controllers/profiles_controller.rb [hunk 1]
```diff
@@ -309,4 +309,8 @@ def filter
 
     render 'profiles/fStudentList'
   end
+
+  def fStudentList
+    @applications = Application.all
+  end
 end
```
Comment (linhpha) [inline app/controllers/profiles_controller.rb]: Can you remove this method? I believe this is from merge conflict so it's redundant. There is already a fStudentList method in profiles_controller

File: app/controllers/application_controller.rb [hunk 2]
```diff
@@ -9,6 +9,12 @@ def configure_permitted_parameters
     devise_parameter_sanitizer.permit(:sign_up, keys: [:first_name, :last_name, :username, :email, :password, :password_confirmation, :id_card, :weblink, :university_id])
   end
 
+  # to override the SessionsController#create and save profile_id to session variable : method 1
+  # def after_sign_in_path_for(resource)
+  #   profile = Profile.joins("INNER JOIN students ON students.id=profiles.student_id AND students.username='#{current_student.username}'")
+  #   session[:current_profile] = profile.id
+  #   root_path
+  # end
```
Comment (linhpha) [inline app/controllers/application_controller.rb]: Please remove this method

File: db/schema.rb [hunk 3]
```diff
@@ -123,20 +123,20 @@
     t.integer  "gre_quant"
     t.decimal  "gre_writing"
     t.integer  "gre_verbal"
-    t.integer  "country_id"
+    t.string   "college"
```
Comment (Avanti0532) [inline db/schema.rb]: Can you regenerate your schema as there are no changes in database fields?

File: app/controllers/application_controller.rb (hunk 2 above)
Comment (juliachaloupka) [reply app/controllers/application_controller.rb]: what method?

File: db/schema.rb (hunk 3 above)
Comment (juliachaloupka) [reply db/schema.rb]: I don't know what you mean, this is the schema that was generated after I merged with master and ran rake db:schema:load

File: app/controllers/profiles_controller.rb (hunk 1 above)
Comment (juliachaloupka) [reply app/controllers/profiles_controller.rb]: done

File: db/schema.rb (hunk 3 above)
Comment (Avanti0532) [reply db/schema.rb]: You can pull from master again and re-run the migration to make sure that your schema is inline with master.

File: app/controllers/application_controller.rb (hunk 2 above)
Comment (linhpha) [reply app/controllers/application_controller.rb]: The commented method  def after_sign_in_path_for(resource). It's not needed anymore

**Pull request merged**